### PR TITLE
Link fixes

### PR DESCRIPTION
**Pull request by RishabhTayal**
fixed documentation links

### CODE REVIEW

File: fastlane/lib/fastlane/actions/create_app_online.rb [hunk 1]
```diff
@@ -30,7 +30,7 @@ def details
           'Create new apps on iTunes Connect and Apple Developer Portal via _produce_.',
           'If the app already exists, `create_app_online` will not do anything.',
           'For more information about produce, visit its GitHub page:',
```
Comment (janpio) [inline fastlane/lib/fastlane/actions/create_app_online.rb]: This is now not true any more. Replace with something more appropriate, like "documentation".

File: fastlane/lib/fastlane/setup/setup_android.rb [hunk 2]
```diff
@@ -23,7 +23,7 @@ def generate_appfile
       puts ""
       puts "To automatically upload builds and metadata to Google Play, fastlane needs a service action json secret file".yellow
       puts "Feel free to just click Enter to skip not provide certain things"
-      puts "Follow the Setup Guide on how to get the Json file: https://github.com/fastlane/fastlane/tree/master/supply#setup".yellow
+      puts "Follow the Setup Guide on how to get the Json file: https://docs.fastlane.tools/actions/supply/".yellow
```
Comment (janpio) [inline fastlane/lib/fastlane/setup/setup_android.rb]: The anchor also exists here: https://docs.fastlane.tools/actions/supply/#setup

File: cert/lib/cert/commands_generator.rb [hunk 3]
```diff
@@ -17,7 +17,7 @@ def run
       program :description, 'CLI for \'cert\' - Create new iOS code signing certificates'
       program :help, 'Author', 'Felix Krause <cert@krausefx.com>'
       program :help, 'Website', 'https://fastlane.tools'
-      program :help, 'GitHub', 'https://github.com/fastlane/fastlane/tree/master/cert#readme'
+      program :help, 'GitHub', 'https://docs.fastlane.tools/actions/cert/'
```
Comment (janpio) [inline cert/lib/cert/commands_generator.rb]: "GitHub" is not correct here then...

**Review by janpio — CHANGES_REQUESTED** [review]
Thanks for tackling this!

I am not sure about the `program :help, 'GitHub' ...` changes though, we will have to decide what to do there.

The docs and comment changes are fine though, just some comment on where there is a better link or surrounding text has to be changed as well.

**Comment by KrauseFx** [comment]
Also there is something wrong with this PR again unfortunately, many unrelated commits here

File: cert/lib/cert/commands_generator.rb (hunk 3 above)
Comment (janpio) [reply cert/lib/cert/commands_generator.rb]: Talked to other contributors, this should probably be changed to "Documentation" on all cases where the Github link was replaced by a `docs.fastlane.tools` one.

**Comment by janpio** [comment]
@RishabhTayal Can you shed some line on all these unconnected commits? I think merging might go well as Github recognizes the real changes you made, but this is always a gamble. 
Can you share your process that you use to create a new branch in that you then commit?

File: fastlane/docs/FAQs.md [hunk 4]
```diff
@@ -1,3 +1,117 @@
-<h3 align="center">
```
Comment (janpio) [inline fastlane/docs/FAQs.md]: This file/change is not wanted here

**Review by janpio — CHANGES_REQUESTED** [review]

File: fastlane/docs/FAQs.md (hunk 4 above)
Comment (RishabhTayal) [reply fastlane/docs/FAQs.md]: I am still trying to figure out how to remove the unwanted commits. I merged `master` into my branch and got these changes. 🤷‍♂️

Comment (janpio) [reply fastlane/docs/FAQs.md]: You will get better results if you rebase your branch onto master. (Meaning: Moving your new commits from where they were onto the most recent commits on master)

Comment (RishabhTayal) [reply fastlane/docs/FAQs.md]: That's what I am trying to do. But when I rebase I get multiple merge conflicts. Here is a screen cap.

https://www.dropbox.com/s/vldkuudauduvvmh/Untitled.mov?dl=0

Comment (janpio) [reply fastlane/docs/FAQs.md]: What are you clicking after you did "Resovle using theirs"? It is enough to have the changes all staged, then you can "Continue to rebase". I think you might be pushing at this moment again which fails?

**Comment by janpio** [comment]
You might have more luck creating a new branch from a current master, then cherry pick your commit you did to fix the links to that and create a new PR that is clean from the start. This one might be a bit messed up already.

**Comment by RishabhTayal** [comment]
Finally created a new pull request with all the commits removed. 😌
https://github.com/fastlane/fastlane/pull/11327

I will close this PR now. Sorry for the inconvenience.

**Pull request closed without merging**